posts/2020-09-29-microprofile-openapi-beta-200011.adoc

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+title: "MicroProfile OpenAPI 2.0 plus new features and loads of improvements for Jakarta EE 9 in Open Liberty 20.0.0.11-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: MicroProfile OpenAPI 2.0 plus new features and loads of improvements for Jakarta EE 9 in Open Liberty 20.0.0.11-beta - OpenLiberty.io
+seo-description: Open Liberty 20.0.0.11-beta comes with MicroProfile OpenAPI 2.0 which allows users to generate documentation directly from APIs, plus two new features for Jakarta EE 9.
+blog_description: Open Liberty 20.0.0.11-beta comes with MicroProfile OpenAPI 2.0 which allows users to generate documentation directly from APIs, plus two new features for Jakarta EE 9.
+---
+= New features and loads of improvements for Jakarta EE 9 plus MicroProfile OpenAPI 2.0 in Open Liberty 20.0.0.11-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+In 20.0.0.11-beta we have a preview of MicroProfile OpenAPI 2.0, which enables you to generate documentation directly from APIs. We have also added two new Jakarta EE 9 features, along with improvements to existing Jakarta EE 9 features, including many moving up to the latest API Release Candidates.
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a new, larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the All Beta Features package or the Jakarta EE 9 Beta Features package, depending on what you want to try).
+
+If you try either package, <<feedback, let us know what you think>>.
+
+[#allbeta]
+== All Beta Features package
+
+
+image::img/blog/logos/home_logo_microprofile.png[MicroProfile Logo,width=70%,align="center"]
```

Comment:
I think move the image down to after the first paragraph (for both MP and Jakarta EE images). So that there's text immediately after the heading, then the image in both cases.

Reply:
Done